ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -1210,60 +1213,25 @@ + (ORKOrderedTask *)PSATTaskWithIdentifier:(NSString *)identifier
-            step.title = ORKLocalizedString(@"TREMOR_TEST_TITLE", nil);
-            step.text = intendedUseDescription ? : ORKLocalizedString(@"TREMOR_TEST_INTENDED_USE", nil);
-            step.image = [UIImage imageNamed:@"tremortest1" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-            step.shouldTintImages = YES;
-            
-            ORKStepArrayAddStep(steps, step);
-        }
-        {
-            ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:ORKInstruction1StepIdentifier];
-            step.title = ORKLocalizedString(@"TREMOR_TEST_TITLE", nil);
-            step.text = ORKLocalizedString(@"TREMOR_TEST_INTRO_2_TEXT", nil);
-            
-            NSArray<NSString *>*detailStringForNumberOfTasks = @[
-                                                                 ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_1_TASK", nil),
-                                                                 ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_2_TASK", nil),
-                                                                 ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_3_TASK", nil),
-                                                                 ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_4_TASK", nil),
-                                                                 ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_5_TASK", nil)
-                                                                 ];
 
-            // start with the count for all the tasks, then subtract one for each excluded task flag
-            static const NSInteger allTasks = 5; // hold in lap, outstretched arm, elbow bent, repeatedly touching nose, queen wave
-            NSInteger actualTasksIndex = allTasks - 1;
-            NSInteger bits = sizeof(activeTaskOptions) * CHAR_BIT; // don't assume exclude flags are sequential--check all positions
-            for (NSInteger i = 0; i < bits ; ++i) {
-                if (activeTaskOptions & (1 << i)) {
-                    actualTasksIndex--;
-                }
-            }
-            
-            NSString *detailFormat = ORKLocalizedString(@"TREMOR_TEST_INTRO_2_DETAIL_%", nil);
-            step.detailText = [NSString stringWithFormat:detailFormat, detailStringForNumberOfTasks[actualTasksIndex]];
-            step.image = [UIImage imageNamed:@"tremortest2" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-            step.shouldTintImages = YES;
-            
-            ORKStepArrayAddStep(steps, step);
-        }
-    }
-    
     if (! (activeTaskOptions & ORKTremorActiveTaskOptionExcludeHandInLap)) {
         if (! (options & ORKPredefinedTaskOptionExcludeInstructions)) {
-            ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"instruction2"];
+            NSString *stepIdentifier = [self stepIdentifier:@"instruction2" withHandIdentifier:handIdentifier];
```

Comment:
Nit: Only just noticed the hardcoded "instruction2". Could you replace that with a predefined key? Thanks.